apps/src/p5lab/drawUtils.js

```diff
@@ -56,37 +56,43 @@ export function speechBubble(
   width,
   height,
   {
-    triangleSize = 10,
-    triangleTipX = x,
-    rectangleCornerRadius = 8,
+    tailSize = 10,
+    tailTipX = x,
+    padding = 8,
     fill = 'white',
     strokeWeight = 2,
     stroke = 'gray'
-  } = {}
+  } = {},
+  style
 ) {
   const minX = x - width / 2;
-  const minY = y - height - triangleSize;
-  const maxY = y - triangleSize;
+  const minY = y - height - tailSize;
+  const maxY = y - tailSize;
+  const tailTopX = (tailTipX + (minX + width / 2)) / 2;
+  const tailBottomX = (tailTipX + tailTopX) / 2;
 
   p5.push();
   p5.stroke(stroke);
   p5.strokeWeight(strokeWeight);
   p5.fill(fill);
-  p5.beginShape();
```

Comment:
I noticed the `beginShape` and `endShape` are missing from the new version - intentional?

Reply:
Intentional. These are both intended to be used with `vertex` or `curveVertex`, neither of which have ever been part of this drawing. I don't think they were needed.
https://p5js.org/reference/#/p5/beginShape